client/src/components/Header/RecentSearches.tsx

```diff
@@ -1,16 +1,81 @@
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import styled from '@emotion/styled';
+import { getLocalStorageItem, removeLocalStorage, removeLocalStorageAll } from '@/utils';
+import { RECENT_SEARCH_KEY } from './utils';
 
 type RecentSearch = {
   name: string;
   date: string;
 };
 
 interface RecentSearchsProps {
-  searches: RecentSearch[];
   isFocused: boolean;
+  removeFocus: () => void;
 }
 
+const RecentSearches = (props: RecentSearchsProps): JSX.Element => {
```

Comment:
```suggestion
const RecentSearches = (props: IRecentSearchesProps): JSX.Element => {
```

interface에 오타도 있고 규칙과 어긋나는 점도 있네욤 👀

Reply:
수정했씁니다 :)